src/displayList.js

```diff
@@ -0,0 +1,41 @@
+import loadImage from './load-image-module';
+
+const displayList = (array) => {
+  console.log('hello');
+  const listContainer = document.getElementById('list-container');
+  listContainer.innerHTML = '';
+
+  for (let j = 0; j < 10; j += 3) {
+    const row = document.createElement('div');
+    row.classList.add('row');
+
+    for (let i = 0; i < 3; i += 1) {
+      const card = document.createElement('div');
+      card.classList.add('col-sm', 'card');
+      card.setAttribute('id', `card-${array[i + j].id}`);
+      card.innerHTML = `
+    <div class="card-img-top"></div>
+    <div class="card-body d-flex flex-column align-items-center gap-1">
+    <div class="card-title w-100 d-flex justify-content-between">  
+    <h5>${array[i + j].name}</h5>
+    <div class="card-text d-flex flex-column align-items-end">
+    <i class="far fa-heart"></i>
+    <p>Likes: 0</p>
```

Comment:
- [ ] I would recommend just putting the **heart image** for likes and then the **number of likes**, removing the "Likes:". It would look more simple and is practical.